.github/workflows/linux.yml

```diff
@@ -24,56 +24,81 @@ on:
       - master
 
 concurrency:
-  group: ${{ github.head_ref || github.run_id }}-linux
+  group: ${{ github.ref || github.run_id }}-linux
   cancel-in-progress: true
 
 jobs:
   Build:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04-8-cores
+    runs-on: aks-linux-16-cores
+    container:
+      image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+      volumes:
+        - /mount/caches:/mount/caches
     env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
       CMAKE_BUILD_TYPE: 'Release'
       CMAKE_GENERATOR: 'Ninja'
       CMAKE_CXX_COMPILER_LAUNCHER: ccache
       CMAKE_C_COMPILER_LAUNCHER: ccache
-      OPENVINO_REPO: ${{ github.workspace }}/openvino
-      OPENVINO_CONTRIB_REPO: ${{ github.workspace }}/openvino_contrib
-      INSTALL_DIR: ${{ github.workspace }}/install
-      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
-      SAMPLES_INSTALL_DIR: ${{ github.workspace }}/install/samples
-      LAYER_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/layer_tests
-      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
-      BUILD_DIR: ${{ github.workspace }}/build
-      OV_TEMP: ${{ github.workspace }}/openvino_temp
+      GITHUB_WORKSPACE: '/__w/openvino/openvino'
```

Comment:
if we decided hardcode those variables is it possible to simplify them (/__w/openvino/openvino/openvino - > __w/openvino)?

Reply:
Let's keep them the same as the content of `${{ github.workspace }}` (this is exact value as it is appears inside a container). It will allow us to switch back to `${{ github.workspace }}` in the future.